cdn-index.template

```diff
@@ -1,5 +1,5 @@
 <html>
-<head><link rel="stylesheet" href="https://static.buildit.digital/gravity/v1.0.1/gravity.css"></head>
+<head><link rel="stylesheet" href="https://static.buildit.digital/gravity-ui-web/latest/gravity.css"></head>
 <body>
 <h1>Versions</h1>
 <ul>{versionlist}</ul>
```

Comment:
Can you please add a link to the file size history page on here. That'll make it easier to find / discover later on. Just chuck it at the bottom of the page for now ;-)